src/ModelComparison.jl

```diff
@@ -98,7 +98,8 @@ function loo_compare(
     )
     # Subtract the effective number of params and elpd ests; leave mcse+pareto_k the same
     @views base_case = pointwise_diffs[:, 1:3, 1]
-    @views @. pointwise_diffs[:, 1:3, Not(1)] = pointwise_diffs[:, 1:3, Not(1)] - base_case
+    lastidx = axes(pointwise_diffs, 3)[2:end]
```

Comment:
This is wrong in general:
```julia
julia> using OffsetArrays

julia> x = OffsetArray(1:7, -3:3)
1:7 with indices -3:3

julia> axes(x, 1)[2:end]
2:3

julia> axes(x, 1)
OffsetArrays.IdOffsetRange(values=-3:3, indices=-3:3)
```

Reply:
Good catch. I haven't worked with OffsetArrays, so I'm not as familiar with other indexing systems.